### ISSUE TITLE
Confuses macOS & iOS/iPadOS apps

### ISSUE THREAD

**Issue by andrewcrook** [mas-cli/mas#330]
## Your Environment

- `mas version`: 1.8.1
- macOS version
  System Version: macOS 11.2.3 (20D91)
  Kernel Version: Darwin 20.3.0

### mas Install Method

`.pkg` installer from [releases](https://github.com/mas-cli/mas/releases)

## Describe the Bug

I installed 1.8.1 and have simular issue as others.

### **mas outdated**

```
1501689671 BASS2 (5.27878.1 -> 1.3.27878)
Warning: Identifier 957810159 not found in store. Was expected to identify Raindrop.io for Safari.
922765270 LiquidText (1.1.8 -> 4.0.12)
405399194 Kindle (1.30.0 -> 1.30)
1107421413 1Blocker (3.14 -> 3.14.1)
```

### **looking at the App Store version info.**
Beyond a Steel Sky 1.3.27878 -   latest installed
Raindrop.io for Safari -  957810159 not found in store
LiquidText Version  Version 1.1.8 - latest installed
Kindle Version 1.30 - latest is installed

**Updated 1Blocker fine **

### **Raindrop  id changed?**
```
mas search 'Raindrop.io for Safari'
1549370672  Save to Raindrop.io (5.0.1)
```

### **Goodnotes does have an a update and didn't show in the  mas outdated  list**

```console
$ mas info 1444383602
GoodNotes 5 5.6.44 [7.99]
By: Time Base Technology Limited
Released: 2021-03-09
Minimum OS: 12.0
Size: 179.2 MB
From: https://apps.apple.com/us/app/goodnotes-5/id1444383602?uo=4
```

latest version in App Store is Version 5.6.46.

I updated from the App Store.

Running `mas info` again:

```
$ mas info 1444383602
GoodNotes 5 5.6.44 [7.99]
By: Time Base Technology Limited
Released: 2021-03-09
Minimum OS: 12.0
Size: 179.2 MB
From: https://apps.apple.com/us/app/goodnotes-5/id1444383602?uo=4
```

mas outdated
```
1501689671 BASS2 (5.27878.1 -> 1.3.27878)
Warning: Identifier 957810159 not found in store. Was expected to identify Raindrop.io for Safari.
1444383602 GoodNotes (5.6.46 -> 5.6.44)
922765270 LiquidText (1.1.8 -> 4.0.12)
405399194 Kindle (1.30.0 -> 1.30)
```

**Not listing the latest version of Goodnotes on App Store which is installed.**

**Comment by lonelykid** [comment]
Duplicate of https://github.com/mas-cli/mas/issues/329

It's better to discuss at one place. This is the limitation of this API provided by Apple.

**Comment by andrewcrook** [comment]
also see #290

**Comment by chris-araman** [comment]
The remaining discrepancies in `outdated` should be resolved by #346.
